shared-bindings/wifi/Radio.c

```diff
@@ -102,6 +102,26 @@ STATIC mp_obj_t wifi_radio_stop_scanning_networks(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(wifi_radio_stop_scanning_networks_obj, wifi_radio_stop_scanning_networks);
 
+//|     def set_hostname(self, hostname: ReadableBuffer) -> None:
```

Comment:
Please do this as a property instead. That way it's simpler to read and write it.